luigi/contrib/s3.py

```diff
@@ -188,7 +188,7 @@ def exists(self, path):
         logger.debug('Path %s does not exist', path)
         return False
 
-    def remove(self, path, recursive=True):
+    def remove(self, path, recursive=True, skip_trash=True):
```

Comment:
What is this?

Reply:
the parent function in FileSystem class has this attribute. Thought will be good to keep compatibility.

Reply:
This looks to be something added to FileSystem for hadoop/hive/webhdfs [long ago](https://github.com/spotify/luigi/pull/680). It serves no purpose for S3 given that there is no "trash" for s3.

It's also worth noting that [LocalFileSystem did not include this arg](https://github.com/spotify/luigi/blob/master/luigi/local_target.py#L94) either.

For that, i'm going to say to exclude `skip_trash` for S3Client's `remove()`.

But nice eye for checking it's parent!